### DISCUSSION TITLE
Set Profile "while held" not working for same button in different profile

### DISCUSSION THREAD

**Discussion by dwi110** [AntiMicroX/antimicrox#1028] [Ideas]
Im having trouble to describe what i want to say :d 

i want to operate 4 cameras and 1 scoreboard with a controller

buttons A B X Y select profile 1 to 4 according to the camera (with press)

but i want in all 4 profiles holding L1 to go to profile 5 (while held) to do some scoreboard actions and then go back to the profile from where it was triggered.

The Problem is, antimicrox always goes back to the profile where i last set "go to profile 5 while held".

Example:

i configured profile 4 L1 button last. Im in Profile 2 and i hold L1 - when i release it is not going back to profile 2 but to Profile 4.

Is this a bug or by design and any idea how i could workaround?

**Comment by dwi110** [comment]
no answers so far :( would also pay a bounty for fixing/adding this feature

**Reply by pktiuk** [reply]
Linked issue: https://github.com/AntiMicroX/antimicrox/issues/1048
